### DISCUSSION TITLE
UX ideas

### DISCUSSION THREAD

**Discussion by markdav-is** [OpenEugene/fire-refuge#8] [Ideas]
Some apps like this take a map-first approach.  I'm a fan of card and lists too.  I really think that a chat style interface might be a real win as well.  Maybe we can approach this with a user centered approach.

**Comment by thomasfiorelli** [comment]
I really like the idea of combining the card/list approach with a map tool somewhere above/below on the page. It would be great if folks could filter the list by search criteria (Ex: I need all hosts willing to accept pets). I'm imagining that someone would click on a card and the map would zoom to the location for that record.
